gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
-#Mon Jul 22 11:10:54 CEST 2019
+#Mon Dec 14 20:31:47 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

You can upgrade to the latest Gradle version if you want
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```